pcsx2/GS/Renderers/DX11/GSDevice11.cpp

```diff
@@ -1347,8 +1347,8 @@ void GSDevice11::UpdateCLUTTexture(GSTexture* sTex, float sScale, u32 offsetX, u
 	// match merge cb
 	struct Uniforms
 	{
-		float scale;
-		float pad1[3];
+		[[maybe_unused]] float scale;
```

Comment:
Can we not do this? It really hurts readability.

None of the fields are "used", because it's just using the struct to pack it into the uniform block layout for the shader. So I'm not sure why you're only marking two of them, and not all. But marking all would be even uglier.